### PR TITLE
WIP: Show tasks and allocations for instances

### PR DESCRIPTION
**Pull request by swsnr**
Corresponding project: https://github.com/FRosner/cluster-broccoli/projects/2

### CODE REVIEW

File: server/src/main/scala/de/frosner/broccoli/nomad/NomadConfiguration.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package de.frosner.broccoli.nomad
+
+import com.typesafe.config.Config
+
+/**
+  * The Nomad configuration.
+  *
+  * @param url The URL to connect to to access nomad via HTTP.
+  */
+final case class NomadConfiguration(url: String)
```
Comment (FRosner) [inline server/src/main/scala/de/frosner/broccoli/nomad/NomadConfiguration.scala]: What happens here if the url is not set?

Comment (swsnr) [reply server/src/main/scala/de/frosner/broccoli/nomad/NomadConfiguration.scala]: In application.conf?  It'll fallback to the one from reference.conf.

Comment (FRosner) [reply server/src/main/scala/de/frosner/broccoli/nomad/NomadConfiguration.scala]: What will happen to the application if neither application.conf nor reference.conf contains this value?

Comment (swsnr) [reply server/src/main/scala/de/frosner/broccoli/nomad/NomadConfiguration.scala]: It'll throw an exception.  reference.conf is supposed to contain all configuration settings you use throughout the application; hence the name.

Comment (swsnr) [reply server/src/main/scala/de/frosner/broccoli/nomad/NomadConfiguration.scala]: PS: reference.conf is not for the user of an application: You shouldn't modify reference.conf during deployment; it's part of the application code essentially

**Comment by FRosner** [comment]
Is the reference.conf packaged inside the jar or the dist zip automatically?

Does play exit cleanly if there is an exception at this point? Will it respect shutdown hooks?

Thanks!

**Comment by swsnr** [comment]
@FRosner  Why should `reference.conf` not be packaged?  It's a resource like any other, isn't it?

I think that Play runs stop hooks no matter how the application exits, but I would not bother to find out in detail.  

As I said `reference.conf` should contain **all** settings of the application; if an exception gets thrown here, that's a programmer error.

**Comment by swsnr** [comment]
@FRosner This is how it looks like with the latest push:

bildschirmfoto 2017-08-02 um 17 28 23

**Comment by swsnr** [comment]
@FRosner I'll close this PR and split it into smaller ones for easier review.

**Pull request closed without merging**